armi/apps.py

```diff
@@ -332,6 +337,11 @@ def __registerUserPluginsInternalImport(self, pluginPath):
         assert issubclass(plugin, plugins.UserPlugin)
         self._pm.register(plugin)
 
+        # ensure UserPlugin flags are loaded
+        newFlags = plugin.defineFlags()
+        if newFlags:
+            Flags.extend(newFlags)
+
```

Comment:
this works. there is a bit of duplicate code here, so maybe eventually we should make a `registerUserPlugin` method where everything from 
assert issubclass(plugin, plugins.UserPlugin)
down goes in